lib/pages/connect_wallet/mobile_ax_wallet_login_page.dart

```diff
@@ -0,0 +1,182 @@
+              margin: const EdgeInsets.only(top: 10),
+              decoration: boxDecoration(
+                Colors.grey.withOpacity(.2),
+                10,
+                1,
+                Colors.grey,
+              ),
+              child: Center(
+                child: TextFormField(
+                  controller: seedPhraseTextController,
+                  maxLines: null,
+                  decoration: const InputDecoration(
+                    hintText: 'Enter your seed phrase here...',
+                    focusedBorder: InputBorder.none,
+                    enabledBorder: InputBorder.none,
+                    contentPadding: EdgeInsetsDirectional.only(start: 10),
+                  ),
+                  style: const TextStyle(
+                    fontSize: 14,
+                    overflow: TextOverflow.ellipsis,
+                  ),
+                  textAlign: TextAlign.center,
+                ),
+              ),
+            ),
+            Container(
+              margin: const EdgeInsets.only(top: 60),
+              width: _width * 0.5,
+              decoration: boxDecoration(
+                Colors.amber[300]!.withOpacity(0.15),
+                20,
+                1,
+                Colors.transparent,
+              ),
+              child: TextButton(
+                onPressed: () {
+                  Navigator.push(
+                    context,
+                    MaterialPageRoute<void>(
+                      builder: (context) => const V1App(),
+                    ),
+                  );
+                },
+                child: Text(
+                  'Continue to App',
+                  style: textStyle(Colors.amber[400]!, 20, true, false),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  TextStyle textStyle(Color color, double size, bool isBold, bool isUline) {
+    // ignore: curly_braces_in_flow_control_structures
+    if (isBold) if (isUline) {
+      return TextStyle(
+        color: color,
+        fontFamily: 'OpenSans',
+        fontSize: size,
+        fontWeight: FontWeight.w400,
+        decoration: TextDecoration.underline,
+      );
+    } else {
+      return TextStyle(
+        color: color,
+        fontFamily: 'OpenSans',
+        fontSize: size,
+        fontWeight: FontWeight.w400,
+      );
+    }
+    else if (isUline) {
+      return TextStyle(
+        color: color,
+        fontFamily: 'OpenSans',
+        fontSize: size,
+        decoration: TextDecoration.underline,
+      );
+    } else {
+      return TextStyle(
+        color: color,
+        fontFamily: 'OpenSans',
+        fontSize: size,
+      );
+    }
+  }
+
+  BoxDecoration boxDecoration(
+    Color col,
+    double rad,
+    double borWid,
+    Color borCol,
+  ) {
+    return BoxDecoration(
+      color: col,
+      borderRadius: BorderRadius.circular(rad),
+      border: Border.all(color: secondaryGreyColor, width: borWid),
+    );
+  }
+}
```

Comment:
At some point we will have to separate the custom styles and decorations but this should do for now

Reply:
Of course, but obviously the point of this PR was to keep refactoring to a minimum level and only address lint errors/warning.